CHANGELOG.md

```diff
@@ -4,6 +4,63 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.57.0]
+
+### Major/Breaking Changes
+* `kani-cov`: A coverage tool for Kani by @adpaco-aws in https://github.com/model-checking/kani/pull/3121
+* Add a timeout option by @zhassan-aws in https://github.com/model-checking/kani/pull/3649
+* Loop Contracts Annotation for While-Loop by @qinheping in https://github.com/model-checking/kani/pull/3151
+* [Breaking change] Make `kani::check` private by @celinval in https://github.com/model-checking/kani/pull/3614
+* Remove symtab json support by @celinval in https://github.com/model-checking/kani/pull/3695
+* Remove CBMC viewer and visualize option by @zhassan-aws in https://github.com/model-checking/kani/pull/3699
+* Dropping support for Ubuntu 18.04 / AL2. by @thanhnguyen-aws in https://github.com/model-checking/kani/pull/3744
+
+### What's Changed
+* Remove the overflow checks for wrapping_offset by @zhassan-aws in https://github.com/model-checking/kani/pull/3589
+* Support fully-qualified --package arguments by @celinval in https://github.com/model-checking/kani/pull/3593
+* Implement proper function pointer handling for validity checks by @celinval in https://github.com/model-checking/kani/pull/3606
+* Add fn that checks pointers point to same allocation by @celinval in https://github.com/model-checking/kani/pull/3583
+* [aeneas] Preserve variable names by @zhassan-aws in https://github.com/model-checking/kani/pull/3560
+* Emit an error when proof_for_contract function is not found by @zhassan-aws in https://github.com/model-checking/kani/pull/3609
+* [Lean] Rename user-facing options from Aeneas to Lean by @zhassan-aws in https://github.com/model-checking/kani/pull/3630
```

Comment:
Should we pick one of [aeneas] or [Lean]? (Maybe just [Lean], given the rename)?

Reply:
Using `[Lean back-end]` throughout now.